doc/nrf/releases_and_maturity/migration/migration_bicr_nRF54H20.rst

```diff
@@ -0,0 +1,470 @@
++======================+======================+======================+
+| P1                   | ``<gpio1 2>``        | EXTERNAL_1V8         |
+| P2                   | ``<gpio2 2>``        | EXTERNAL_1V8         |
+| P6                   | ``<gpio6 2>``        | EXTERNAL_1V8         |
+| P7                   | ``<gpio7 2>``        | EXTERNAL_1V8         |
+| P9                   | ``<gpio9 4>``        | EXTERNAL_FULL        |
++----------------------+----------------------+----------------------+
+
+The resulting JSON configuration is structured as follows:
+
+.. code-block:: json
+
+   {
+     "ioPortPower": {
+       "p1Supply": "EXTERNAL_1V8",
+       "p2Supply": "EXTERNAL_1V8",
+       "p6Supply": "EXTERNAL_1V8",
+       "p7Supply": "EXTERNAL_1V8",
+       "p9Supply": "EXTERNAL_FULL"
+     }
+   }
+
+GPIO power drive and impedance configuration
+============================================
+
+This DTS option configures the ``BICR`` → ``IOPORT.DRIVECTRL0`` register:
+
+.. code-block:: dts
+
+   ioport-drivectrls = <&gpio6 50>, <&gpio7 50>;
+
+This section specifies the IO port impedance settings for **P6** and **P7**.
+
+As with the GPIO power configuration, each GPIO reference indicates the corresponding port number, while the second value indicates the port's impedance in Ohms.
+
+In this example, both **P6** and **P7** are configured to use a 50 Ohm impedance.
+
+The nRF54H20 SoC allows you to select from these supported impedance values:
+
+* 33 Ohms
+* 40 Ohms
+* 50 Ohms
+* 66 Ohms
+* 100 Ohms
+
+The DTS configuration described in the nRF54H20 DK files can be represented in JSON as follows:
+
+.. code-block:: json
+
+   {
+     "ioPortImpedance": {
+       "p6ImpedanceOhms": 50,
+       "p7ImpedanceOhms": 50
+     }
+   }
+
+Low Frequency Oscillator (LFOSC) configuration
+==============================================
+
+The following DTS options configure parts of the ``BICR`` → ``LFOSC.LFXOCONFIG`` register:
+
+.. code-block:: dts
+
+   lfosc-mode = "crystal";
+   lfosc-loadcap = <15>;
+
+The following elements of the ``LFOSC.LFXOCONFIG`` register are defined in the BICR DTS format file:
+
+* ACCURACY (set using ``lfosc-mode``)
+* MODE (``lfosc-mode``)
+* LOADCAP (``lfosc-loadcap``)
+* TIME (``lfosc-startup``)
+
+The available options for these settings are provided in the following tables.
+
+The JSON format is inside ``lfosc: { lfxo: { ... } }``.
+
+ACCURACY
+--------
+
++----------------------+--------------------+
+| DTS (lfosc-accuracy) | JSON (accuracyPPM) |
++======================+====================+
+| 20                   | 20                 |
+| 30                   | 30                 |
+| 50                   | 50                 |
+| 75                   | 75                 |
+| 100                  | 100                |
+| 150                  | 150                |
+| 250                  | 250                |
+| 500                  | 500                |
++----------------------+--------------------+
```

Comment:
The rows in this and several of the other tables are being collapsed into a single row in the rendered doc